### PR TITLE
tweak default sync settings

### PR DESCRIPTION
**Pull request by lezama**
Let's make Sync Settings less conservative.

p2-poqVs-nS7#comment-124672

What else?

### CODE REVIEW

File: packages/sync/src/class-defaults.php [hunk 1]
```diff
@@ -1160,7 +1160,7 @@ public static function is_multi_network() {
 	 *
 	 * @var int Number of seconds.
 	 */
-	public static $default_cron_sync_time_limit = 30; // 30 seconds.
+	public static $default_cron_sync_time_limit = 4 * MINUTE_IN_SECONDS; // 30 seconds.
```
Comment (gravityrail) [inline packages/sync/src/class-defaults.php]: You need to fix or remove this comment :)

```php
// 30 seconds.
```

**Review by gravityrail — DISMISSED** [review]
LGTM, just one minor comment.

**Review by nickdaugherty — DISMISSED** [review]
LGTM 🎉

File: packages/sync/src/class-defaults.php (hunk 1 above)
Comment (lezama) [reply packages/sync/src/class-defaults.php]:
```suggestion
	public static $default_cron_sync_time_limit = 4 * MINUTE_IN_SECONDS;
```

**Review by enejb — APPROVED** [review]
:+1: This helps make the cron syncing work better.

**Comment by lezama** [comment]
@jeherve could we add this one to the sync branch?

**Comment by lezama** [comment]
Thanks 😊

**Comment by jeherve** [comment]
Cherry-picked to `branch-8.sync` in 0df8477c9bb24092adf8ede5ba8f452d6223f7e0

**Pull request merged**